src/main.ts

```diff
@@ -130,12 +131,38 @@ async function resolveConfigs(
     logger.error(chalk.red('You must provide at least 1 prompt'));
     process.exit(1);
   }
+
+  // Convert providers to array if not already an array
+  if (config.providers && typeof config.providers === 'string') {
+    config.providers = [config.providers];
+  }
```

Comment:
Is this the right place for this?  It means CLI (`main`) will be inconsistent  with package (`index`)